src/matrix.rs

```diff
@@ -76,11 +78,47 @@ pub fn matrix_inverse<T: Float>(a: &Mat3<T>) -> Mat3<T> {
     [d0/det, -d3/det, d6/det, -d1/det, d4/det, -d7/det, d2/det, -d5/det, d8/det]
 }
 
+///Geneartes to Srgb to Xyz transformation matrix for the given white point
+pub fn rgb_to_xyz_matrix<S: RgbSpace, T: Float>() -> Mat3<T> {
```

Comment:
```rgb_to_xyz_matrix``` and ``` mat3_from_primaries``` should be default implemented in the RgbSpace trait. Then we will be able to manually add the matrix for known conversions like Srgb to Xyz, without recalculating the transform matrix every time.